Nautilus/Handlers/ModDataBankHandler.cs

```diff
@@ -0,0 +1,114 @@
+using BepInEx;
+using Nautilus.Utility;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Nautilus.Handlers;
+public class ModDataBankHandler
+{
+    private static bool isinit;
+    private static List<PDAEncyclopedia.EntryData> waitlist = new();
+    internal static void Initialize(uGUI_EncyclopediaTab tab)
+    {
+        LanguageHandler.SetLanguageLine("EncyPath_Mods", "Mods");
+        isinit = true;
+        foreach(var data in waitlist)
+        {
+            CompleteRegister(data);
+        }
+    }
+
+    private static void CompleteRegister(PDAEncyclopedia.EntryData data)
+    {
+        if (isinit)
+        {
+            InternalLogger.Info($"{data.key} entry added.");
+            PDAHandler.AddEncyclopediaEntry(data);
+            PDAEncyclopedia.Add(data.key, true);
```

Comment:
Does not compile in Below Zero

Reply:
I don't own Below Zero, so I am not able to fix that.

Reply:
The NuGet packages provide all needed dependencies.

Reply:
Should be fixed.